lib/components/UploadPicker.vue

```diff
@@ -280,6 +280,11 @@ export default {
 			this.uploadManager.destination = destination
 			this.newFileMenuEntries = getNewFileMenuEntries(destination)
 		},
+
+		onActionClick(action) {
+			const navigation = getNavigation()
+			action.handler(this.destination, navigation?.active)
```

Comment:
How does it work on the file picker?
I'm guessing the navigation will cause issues here?

Reply:
Maybe the current directory is enough and we don't need the navigation :thinking: 
Actions can also get it themselves then?

Reply:
Yeah, let's remove this: https://github.com/nextcloud-libraries/nextcloud-files/pull/742